sagemaker-featurestore/feature_store_object_detection_ground_truth.ipynb

```diff
@@ -0,0 +1,497 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    s3_bucket_name = sagemaker_session.default_bucket() #This is the bucket for your Feature Store.
</pre><p>I would prefer to replace Feature Store with Offline Store</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2822/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>